src/components/ComparisonDetailMap/ComparisonDetailMap.vue

```diff
@@ -63,23 +75,44 @@
         }
       },
 
-      isLoadingSatelliteImage (newVal) {
-        this.$emit('loading', newVal)
+      extraBuffer () {
+        if (this.map) {
+          this.addSatelliteImageToMap()
+        }
+      },
+
+      generatingSatelliteImageUrl: {
+        handler (newVal) {
+          this.$emit('generatingSatelliteImage', newVal)
+        },
+        deep: true,
       },
     },
 
     methods: {
       async addSatelliteImageToMap () {
-        this.isLoadingSatelliteImage = true
+        this.generatingSatelliteImageUrl.loading.state = true
         const geometry = {
           ...this.reservoirs,
           properties: {
             t: Math.floor(this.date.getTime() / 1000),
           },
         }
 
-        const data = await this.$repo.image.getSatelliteImage(geometry)
+        const data = await this.$repo.image.getSatelliteImage(geometry, this.extraBuffer)
+
+        if (data.error) {
+          this.generatingSatelliteImageUrl.error.state = true
+          this.generatingSatelliteImageUrl.loading.state = false
+          this.generatingSatelliteImageUrl.error.message = `Failed generating satellite images for reservoir. ${this.extraBuffer > 0 ? 'Try choosing a smaller buffer around the reservoir.' : ''}`
```

Comment:
`Try choosing a smaller buffer around the reservoir` only appears if the user has selected a buffer. Otherwise it can be confusing if the satellite images fail and they hadn't even seen this feature, this can happen on larger reservoirs.